machine_management/user_infra/adding-compute-user-infra-general.adoc

```diff
@@ -0,0 +1,32 @@
+[id="adding-compute-user-infra-general"]
+= Adding compute machines to clusters with user-provisioned infrastructure
+include::modules/common-attributes.adoc[]
+:context: adding-compute-user-infra-general
+
+toc::[]
+
+In addition to adding compute machines to a cluster during the user-provisioned infrastructure installation process, you can add more compute machines to a cluster with user-provisioned infrastructure. The process requires some of the same configuration files and parameters used during installation.
+
+== Adding compute machines to Amazon Web Services
+
+To add more compute machines to your {product-title} cluster on Amazon Web Services (AWS), see xref:../../machine_management/user_infra/adding-aws-compute-user-infra.adoc#adding-aws-compute-user-infra[Adding compute machines to AWS by using CloudFormation templates].
+
+== Adding compute machines to Microsoft Azure
+
+To add more compute machines to your {product-title} cluster on Microsoft Azure, see xref:../../installing/installing_azure/installing-azure-user-infra.adoc#installation-creating-azure-control-plane_installing-azure-user-infra[Creating the control plane machines in Azure] in the _Installing a cluster on Azure using ARM templates_ documentation.
+
+== Adding compute machines to Azure Stack Hub
+
+To add more compute machines to your {product-title} cluster on Azure Stack Hub, see xref:../../installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc#installation-creating-azure-control-plane_installing-azure-stack-hub-user-infra[Creating the control plane machines in Azure Stack Hub] in the _Installing a cluster on Azure Stack Hub using ARM templates_ documentation.
```

Comment:
similar question about the ARM templates

Reply:
same :)